converters/toZigbee.js

```diff
@@ -28,6 +28,13 @@ const manufacturerOptions = {
     viessmann: {manufacturerCode: herdsman.Zcl.ManufacturerCode.VIESSMAN_ELEKTRO},
 };
 
+function normalizeCelsiusVersionOfFahrenheit(value) {
```

Comment:
Please move this function to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/lib/utils.js

Reply:
Done.